### PR TITLE
Feature/apply standard js before commit

### PR DESCRIPTION
**Pull request by baodoanblueink**

### CODE REVIEW

File: src/blueink/subclients/person.js [hunk 1]
```diff
@@ -1,72 +1,72 @@
...
          * Create a person using PersonHelper convenience object.
          * @param {object} personHelper PersonHelper setup of a person
          * @returns New Person data
          */
-        createFromPersonHelper: function (personHelper) {
-            return this.create(personHelper.asDict())
-        }, 
+    createFromPersonHelper: function (personHelper) {
+      return this.create(personHelper.asDict())
+    },
```
Comment (trungthanhtong) [inline src/blueink/subclients/person.js:21]: Change `asDict` to `asData`

File: src/example/create_bundle_from_b64.js [hunk 2]
```diff
@@ -1,117 +1,123 @@
-require('dotenv/config');
-const chalk = require('chalk');
-const inquirer = require('inquirer');
-const fetch = require('node-fetch');
+require('dotenv/config')
+const chalk = require('chalk')
+const inquirer = require('inquirer')
+const fetch = require('node-fetch')
```
Comment (trungthanhtong) [inline src/example/create_bundle_from_b64.js:4]: Sort imports

File: src/example/interact_persons.js [hunk 3]
```diff
@@ -1,134 +1,134 @@
-require("dotenv/config");
-const inquirer = require("inquirer");
-const chalk = require("chalk");
-const { Client, PersonHelper } = require("../../index");
+require('dotenv/config')
+const inquirer = require('inquirer')
+const chalk = require('chalk')
+const { Client, PersonHelper } = require('../../index')
```
Comment (trungthanhtong) [inline src/example/interact_persons.js:4]: Reorder imports

File: src/example/create_bundle_from_url.js [hunk 4]
```diff
@@ -1,88 +1,89 @@
-require('dotenv/config');
-const inquirer = require('inquirer');
-const chalk = require('chalk');
+require('dotenv/config')
+const inquirer = require('inquirer')
+const chalk = require('chalk')
```
Comment (trungthanhtong) [inline src/example/create_bundle_from_url.js:3]: Reorder imports

File: src/example/create_bundle_from_template.js [hunk 5]
```diff
@@ -1,92 +1,92 @@
-require('dotenv/config');
-const chalk = require('chalk');
-const inquirer = require('inquirer');
+require('dotenv/config')
+const chalk = require('chalk')
+const inquirer = require('inquirer')
```
Comment (trungthanhtong) [inline src/example/create_bundle_from_template.js:3]: Reorder imports

File: src/example/create_bundle_from_path.js [hunk 6]
```diff
@@ -1,88 +1,88 @@
-require('dotenv/config');
-const inquirer = require('inquirer');
-const chalk = require('chalk');
+require('dotenv/config')
+const inquirer = require('inquirer')
+const chalk = require('chalk')
```
Comment (trungthanhtong) [inline src/example/create_bundle_from_path.js:3]: Reorder imports

File: src/example/interact_webhooks.js [hunk 7]
```diff
@@ -1,124 +1,124 @@
-require("dotenv/config");
-const inquirer = require("inquirer");
-const chalk = require("chalk");
-const { Client } = require("../../index");
-const { EVENT_TYPE } = require("../blueink/constants.js");
+require('dotenv/config')
+const inquirer = require('inquirer')
+const chalk = require('chalk')
+const { Client } = require('../../index')
+const { EVENT_TYPE } = require('../blueink/constants.js')
```
Comment (trungthanhtong) [inline src/example/interact_webhooks.js:5]: Reorder imports

File: src/example/list_bundles.js [hunk 8]
```diff
@@ -1,75 +1,78 @@
-require("dotenv/config");
-const inquirer = require("inquirer");
-const chalk = require("chalk");
-const { Client } = require('../../index');
+require('dotenv/config')
+const inquirer = require('inquirer')
+const chalk = require('chalk')
+const { Client } = require('../../index')
```
Comment (trungthanhtong) [inline src/example/list_bundles.js:4]: Reorder imports

File: src/test/bundle.test.js [hunk 9]
```diff
@@ -1,30 +1,30 @@
-require('dotenv/config');
-const { Client } = require('../../index.js');
+require('dotenv/config')
+const { Client } = require('../../index.js')
```
Comment (trungthanhtong) [inline src/test/bundle.test.js:2]: Imports

File: src/test/pagination.test.js [hunk 10]
```diff
@@ -1,37 +1,39 @@
-require('dotenv/config');
-const { Client } = require('../../index');
+require('dotenv/config')
+const { Client } = require('../../index')
```
Comment (trungthanhtong) [inline src/test/pagination.test.js:2]: Imports

File: src/test/person.test.js [hunk 11]
```diff
@@ -1,79 +1,79 @@
-require('dotenv/config');
-const { Client } = require('../../index');
+require('dotenv/config')
+const { Client } = require('../../index')
```
Comment (trungthanhtong) [inline src/test/person.test.js:2]: Imports

File: src/test/template.test.js [hunk 12]
```diff
@@ -1,18 +1,17 @@
-require('dotenv/config');
-const { Client, BundleHelper } = require('../../index');
+require('dotenv/config')
+const { Client } = require('../../index')
```
Comment (trungthanhtong) [inline src/test/template.test.js:2]: Imports

**Review by trungthanhtong — CHANGES_REQUESTED** [review]
Please see comments. 
We also need to change the names of `PersonHelper.js` and `RequestHelper.js` to `personHelper.js` and `requestHelper.js` for consistent

File: src/blueink/helper/PersonHelper.js [hunk 13]
```diff
@@ -10,116 +10,116 @@ class PersonHelper {
...
+    }
+    let outDict = { ...personOut }
 
     // Merge in the additional data
-    outDict = { ...outDict, ...additionalData };
+    outDict = { ...outDict, ...additionalData }
 
-    return outDict;
+    return outDict
   }
```
[the remark this answers is not in the document]
Comment (geogredoansg) [reply src/blueink/helper/PersonHelper.js:122]: In this case, it is no need to define the additionalData as a function parameter, right?

File: src/blueink/helper/bundleHelper.js [hunk 14]
```diff
@@ -1,417 +1,426 @@
...
+const { fileFromPathSync } = require('formdata-node/file-from-path')
+const get = require('lodash.get')
+const has = require('lodash.has')
+const isEmpty = require('lodash.isempty')
+const { Readable } = require('stream')
 
-const { generateKey } = require('../../util/utility.js');
-const { FIELD_KIND } = require('../constants.js');
+const { generateKey } = require('../../util/utility.js')
+const { FIELD_KIND } = require('../constants.js')
```
Comment (trungthanhtong) [inline src/blueink/helper/bundleHelper.js:10]: Please move constant above util

File: src/example/create_bundle_from_b64.js [hunk 15]
```diff
@@ -1,117 +1,123 @@
-require('dotenv/config');
-const chalk = require('chalk');
-const inquirer = require('inquirer');
-const fetch = require('node-fetch');
+require('dotenv/config')
+const chalk = require('chalk')
+const fetch = require('node-fetch')
+const inquirer = require('inquirer')
```
Comment (trungthanhtong) [inline src/example/create_bundle_from_b64.js:4]: Reorder imports

**Review by trungthanhtong — CHANGES_REQUESTED** [review]
Please see comments

**Comment by zlove** [comment]
This is included in https://github.com/blueinkhq/blueink-client-js/pull/22

**Pull request closed without merging**